assets/src/hooks/useDetour.ts

```diff
@@ -139,7 +152,11 @@ export const useDetour = ({ routePatternId, shape }: OriginalRoute) => {
 
   return {
     /** The current state of the detour machine */
-    state,
+    state: snapshot.matches({ "Detour Drawing": "Editing" })
+      ? DetourState.Edit
+      : snapshot.matches({ "Detour Drawing": "Share Detour" })
+      ? DetourState.Finished
+      : DetourState.Edit,
```

Comment:
Any desires for a follow up PR to get rid of the `DetourState` type?

Reply:
Yes please!

Reply:
Not urgent though IMO

Reply:
Oh especially because `DetourState` is what `DiversionPage` uses to figure out which panel to plant on the left... it's not as immediately simple as it looks O_o

Reply:
Hmm, we could "just do it ™️" in this PR by merging this into this PR https://github.com/mbta/skate/pull/2615

Reply:
I vote nay on that - I think #2615 deserves to be its own chunk of work and should be reviewed separately.